docs/eventlisteners.md

```diff
@@ -9,6 +11,10 @@ metadata:
   name: listener
   namespace: tekton-pipelines
 spec:
-  triggerbindingrefs:
-    - name: pipeline-binding
+  triggers:
+    - triggerBinding:
+        name: pipeline-binding
+      triggerTemplate:
+        name: pipeline-template
+      serviceAccountName: default
```

Comment:
putting `serviceAccount` here makes sense to me!

btw for some reason we call this `serviceAccount` in pipelines so would be good to use consistent naming if we can (e.g. in taskrun: https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md#service-account)

Reply:
ugh but it's called `serviceAccountName` in the rest of kubernetes isn't it :S

![image](https://user-images.githubusercontent.com/432502/63193455-85afb880-c03b-11e9-92d0-327185497810.png)

maybe we should leave this and change pipelines after all

Reply:
Created relevant pipeline issue [here](https://github.com/tektoncd/pipeline/issues/1219) and resolving this conversation.